rollup.config.js

```diff
@@ -2,51 +2,59 @@ import buble from 'rollup-plugin-buble';
 import json from 'rollup-plugin-json';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
-import pkg from './package.json';
+const pkg = require('./package.json');
 
-const config = {
-	input: 'src/index.js',
-	plugins: [
-		json(),
-		commonjs(),
-		buble({
-			target: {
-				node: 4
-			},
-			include: [
-				'src/**',
-				'node_modules/acorn-object-spread/**',
-				'node_modules/unicode-loose-match/**',
-				'node_modules/regexpu-core/**'
-			],
-			transforms: {
-				dangerousForOf: true
-			}
-		}),
-		resolve()
-	],
-	name: 'buble',
-	sourcemap: true
+const createConfig = ({
+	browser = false,
+	external = ['acorn', 'magic-string'],
+	output,
+} = {}) => {
+	return {
+		input: 'src/index.js',
+		output,
+		external,
+		plugins: [
+			json(),
+			commonjs(),
+			buble({
+				target: !browser ? { node: 4 } : null,
+				include: [
+					'src/**',
+					'node_modules/acorn-object-spread/**',
```

Comment:
This is unused.

Reply:
removed this extra acorn-related dep from the include list